parsec/backend/cli/run.py

```diff
@@ -471,8 +497,11 @@ async def _serve_client(stream):
                 await trio.serve_tcp(_serve_client, port, host=host)
 
         click.echo(
-            f"Starting Parsec Backend on {host}:{port} (db={config.db_type}, "
-            f"blockstore={config.blockstore_config.type})"
+            f"Starting Parsec Backend on {host}:{port}"
+            f" (db={config.db_type}"
+            f" blockstore={config.blockstore_config.type}"
+            f" backend_addr={config.backend_addr}"
+            f" email_config={email_config.__class__.__name__})"
```

Comment:
There might be a better way of representing the email config? Maybe use `str(email_config)` and define the `__str__` method for `SmtpEmailConfig` et `MockedEmailConfig`?

Reply:
Done :)